assets/settings/default.json

```diff
@@ -630,18 +630,18 @@
     // Default width of the notification panel.
     "default_width": 380
   },
-  "assistant": {
+  "agent": {
     // Version of this setting.
     "version": "2",
```

Comment:
Shouldn't this version be 1? Given that a new config altogether.

Reply:
It's not really a new config, we're just renaming this key from `assistant` to `agent`---the settings within the key are the same as before!